app/extensions/acm/__init__.py

```diff
@@ -35,12 +37,79 @@ class ACMManager(RestManager):
         'version': {
             'dict': '//version/',
         },
+        'passthrough': {
+            'data': '',
+        },
+        'job': {
+            'detect_request': '//engine/detect/%s',
+            'identification_request': '//engine/identify/%s',
+            'response': '//engine/job/metadata/?jobid=%s'
```

Comment:
is this a new/different url?  (or placeholder?) ... i think in old-world we would use `/api/engine/job/result/?jobid=` so just kinda curious.

Reply:
I've been directed to the metatdata version, possibly by you, at some point for something else. 
This is almost certainly my mistake. The only bit in here that's tested so far is the detect_request.
I suspect the identify is also wrong